tissuelab/gui/vtkviewer/point_editor.py

```diff
@@ -40,8 +40,15 @@ def __init__(self, parent=None, world=None, world_object=None):
         self.selected_axis = 'x'
         self.grab_mode = False
 
-        self.world_object = world_object
-        self.data = deepcopy(world_object.data)
+        if world is None:
+            self.world = World()
+        else:
+            self.world = world
+
+        self.world_object = self.world[points_name]
+        self.data = deepcopy(self.world[points_name].data)
+
+        self.image_world_object = self.world[image_name]
```

Comment:
I would write instead to not dereference several times:
world_object = self.world_object = self.world[points_name]
self.data = deepcopy(world_object)